canon_host/Cargo.toml

```diff
@@ -13,9 +13,8 @@ license = "MPL-2.0"
 lazy_static = "1.4"
 parking_lot = "0.11"
 wasmi = "0.6.0"
-canonical = { path = "../canon", version = "0.4.0" }
-canonical_derive = { path = "../canon_derive", version = "0.4.0" }
+canonical = { path = "../canon", features = ["host"] }
```

Comment:
Shouldn't this point to `0.4` since it's going to be published?

Reply:
adressed